src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets

```diff
@@ -39,12 +39,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PackDependsOn>$(BeforePack); _IntermediatePack; GenerateNuspec; $(PackDependsOn)</PackDependsOn>
     <IsInnerBuild Condition="'$(TargetFramework)' != '' AND '$(TargetFrameworks)' != ''">true</IsInnerBuild>
     <NoBuild Condition="'$(GeneratePackageOnBuild)' == 'true'">true</NoBuild>
+    <SymbolPackageFormat Condition="'$(SymbolPackageFormat)' == ''">symbols.nupkg</SymbolPackageFormat>
```

Comment:
no. we will change the default in 5.0 at some time.

Reply:
The spec needs to be aligned to the behavior then. 
The spec says otherwise https://github.com/NuGet/Engineering/blob/master/Client.Specs/PackageDebugging.md#package-authoring-experience